README.md

```diff
@@ -61,25 +61,26 @@ You can also make params optional by adding a `?` to it.
 
 ### Lazy Loading
 
-Lazy loading (code splitting) with `preact-router` can be implemented easily using the [AsyncRoute](https://www.npmjs.com/package/preact-async-route) module:
-
+Lazy loading (code splitting) with `preact-router` can be implemented using the `lazy`-component.
 ```js
-import AsyncRoute from 'preact-async-route';
-<Router>
-  <Home path="/" />
-  <AsyncRoute
-    path="/friends"
-    getComponent={ () => import('./friends').then(module => module.default) }
-  />
-  <AsyncRoute
-    path="/friends/:id"
-    getComponent={ () => import('./friend').then(module => module.default) }
-    loading={ () => <div>loading...</div> }
-  />
-</Router>
+import { lazy, Suspense } from "preact/compat";
+import { Router } from "preact-router";
+
+const Home = lazy(() => import("./home"));
+
+function App() {
+  return (
+    <div>
+      <Suspense fallback={<div>Loading...</div>}>
```

Comment:
I'm a little wary of recommending this pattern. AsyncRoute transitions between two routes seamlessly by leveraging the fact that preact-router doesn't destroy and recreate routes if they're the same component. With `lazy()`, clicking a link immediately shows the fallback content ("loading..."), then flashes to the new route once loaded - fairly expensive.

Here's a demo showing the effect:
https://codesandbox.io/s/brave-sinoussi-bitsq?file=/components/app.js

Reply:
I don't agree with you. That happens because of naive component that we are providing ("loading...") completely different from the rest of the app causing the flash effect. if we provide a component congruent with the rest of the app i don't think it would be a problem. Besides suspense/code splitting this way is now recommended by React am I wrong?

Reply:
> Besides suspense/code splitting this way is now recommended by React am I wrong?

We're not React and we arguably have a stronger focus on performance. Suspense isn't good for code-splitting as it will very likely lead to wasteful rendering/diffing, that will be immediately thrown out. The concept itself is flawed, which is why we don't recommend Suspense to be used in production.

Reply:
`AsyncRoutes` says [it is deprecated and we should use Suspense/lazy instead](https://github.com/prateekbh/preact-async-route#deprecation-notice)...
@marvinhagemeister you recommend not to use lazy/suspense...
It feels like a vicious circle, isn't it ? 😅